src-backbone/app/js/views/auth/resetPasswordCompleteView.js

```diff
@@ -0,0 +1,69 @@
+const App = require('utils/sanaAppInstance');
+const Helpers = require('utils/helpers');
+
+module.exports = Marionette.ItemView.extend({
+
+    initialize: function(reset_token) {
```

Comment:
There is no real convention but `initialize` at the beginning makes more sense. It is also how objects are defined on Backbone's site.